openfisca_us/parameters/hhs/msp/qmb/income/couple.yaml

```diff
@@ -0,0 +1,15 @@
+description: Monthly maximum income for an eligible individual with an eligible spouse to qualify for the Qualified Medicare Beneficiary (QMB) Program, one of the four Medicare Savings Programs that helps pay for Medicare Part A premiums, Part B premiums, and deductibles, coinsurance, and copayments.
+
+values:
+  2021-01-01: 1430
+  2022-01-01: 1546
+
+metadata:
+  label: Monthly maximum income for an eligible individual with an eligible spouse to qualify for the Qualified Medicare Beneficiary (QMB) Program.
+  name: msp_qmb_income_couple
+  unit: currency-USD
+  reference:
```

Comment:
Could you add a `period: month` line here? It'd enable PolicyEngine to show £1,430/month rather than defaulting to /year.